src/Sentry.AspNetCore/Extensions/HttpContextExtensions.cs

```diff
@@ -37,11 +61,25 @@ internal static class HttpContextExtensions
             var action = routeData?.Values["action"]?.ToString();
             var area = routeData?.Values["area"]?.ToString();
 
-            if (!string.IsNullOrWhiteSpace(action))
+            if (string.IsNullOrWhiteSpace(action) is false)
             {
-                return !string.IsNullOrWhiteSpace(area)
-                    ? $"{area}.{controller}.{action}"
-                    : $"{controller}.{action}";
+                var builder = new StringBuilder();
+                if (context.Request.PathBase.HasValue)
+                {
+                    builder.Append(context.Request.PathBase.Value?.TrimStart('/'))
+                        .Append('.');
+                }
+
+                if (string.IsNullOrWhiteSpace(area) is false)
```

Comment:
Feels a bit verbose to do `is false` here
```suggestion
                if (!string.IsNullOrWhiteSpace(area))
```